src/signals/incident/components/form/MultiTextInput/MultiTextInput.tsx

```diff
@@ -0,0 +1,111 @@
+import type { FunctionComponent } from 'react';
+import React from 'react';
+import styled from 'styled-components';
+import map from 'lodash.map';
+
+import Input from 'components/Input';
+import Button from 'components/Button';
+
+import './style.scss';
```

Comment:
Can we take this opportunity to remove the scss file? It only applies a margin-bottom styling

Reply:
Sure thing!!